.env

```diff
@@ -1 +0,0 @@
-OPENWEATHERMAP_API_KEY=d783fa4b6963f8779267128c0c7cddb1
```

Comment:
without .env file your api key cannot be stored. please verify whether you have added that.